src/message/index.ts

```diff
@@ -41,9 +41,9 @@ export interface Headers extends IncomingHttpHeaders {
  * @property {@linkcode Headers} `headers` - the headers for the event Message
  * @property string `body` - the body of the event Message
```

Comment:
Looks like these docs should be updated with the code change.

Maybe other docs too.